### ISSUE TITLE
[BUG] Bug in quaternion_to_axis_angle

### ISSUE THREAD

**Issue by nkolot** [arraiyopensource/torchgeometry#84]
There is a bug in quaternion_to_axis_angle when the input quaternion is the quaternion corresponding to a 0 rotation ([1,0,0,0]). Specifically the problem is introduced by the division at line 408 in conversions.py

Minimal example to reproduce it
```
import torch
import torchgeometry as tgm
x = torch.tensor([1,0,0,0]).float()
aa = tgm.quaternion_to_angle_axis(x)
```

**Comment by edgarriba** [comment]
@nkolot thanks for the report :D

I link here the code just to make it more visible: https://github.com/arraiyopensource/torchgeometry/blob/master/torchgeometry/core/conversions.py#L408

My guess is that this division needs an epsilon value. Actually, we were discussing similar issues with @versatran01. He's the opinion that should be user that provides an input with non-absolute zeroed values. I'm not quiet sure yet because this could lead to never-ending tickets if people is not aware of this when they build applications. In addition, this could lead to produce silent backprop bugs which I experiencied in the past and is extremely annoying. 

To summarise: it seems lile a quick a fix. If you feel that you can handle it, just go ahead. Otherwise, I'll add it to my to-do list for the v0.1.2 release.

**Comment by versatran01** [comment]
This function looks really complicated, what is the purpose of the mask in this function.

**Comment by edgarriba** [comment]
@versatran01 I didn't implement this function. But I agree, it can be simplified.
